src/screens/Settings/components/Tools/index.tsx

```diff
@@ -43,10 +44,12 @@ export default function Tools({ wineVersion, winePrefix, appName }: Props) {
 
   const handleRunExe = async () => {
     let exe = ''
+    const gameinfo = await getGameInfo(appName)
     const { path } = await ipcRenderer.invoke('openDialog', {
       buttonLabel: t('box.select.button', 'Select'),
       properties: ['openFile'],
-      title: t('box.runexe.title')
+      title: t('box.runexe.title'),
+      defaultPath: gameinfo.install.install_path
```

Comment:
This might not be helpful, usually (at least when I use this feature) I want to run an EXE that I got from somewhere (DirectX installers, .NET, things like that)
I think it also sends the wrong message. If you want to run a different executable as the game, you should use the alternative EXE option and not this.

Reply:
Yes, wasn't sure about this one but I remember that most games comes with a redist folder with the dependencies so then people might actually look there first. If not they can choose a different folder just like it's today.
What do you think?

Reply:
You're right, doesn't hurt to start there. If they really need another EXE, the home folder is one click away (while getting to the game folder from Home is a bit more effort)